build.gradle

```diff
@@ -97,6 +97,15 @@ tasks.register('createReleaseTag', CreateGitTag) {
 
 tasks.named('githubRelease') {
     dependsOn('createReleaseTag')
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.named('publishPlugins') {
+    notCompatibleWithConfigurationCache("$name task does not support configuration caching")
+}
+
+tasks.withType(Sign) {
```

Comment:
Made this more generic to account for upcoming changes in the `plugin-publish` plugin.